.github/workflows/ci.yml

```diff
@@ -21,7 +21,6 @@ jobs:
         uses: actions/checkout@v3
         with:
           submodules: true
-          persist-credentials: false
```

Comment:
It seems this was setup when we initially created the CI. I don't think we need this as we don't have it on the CLI repo?